src/locales/en/pages.yaml

```diff
@@ -158,15 +158,15 @@ contact:
 
     ### Board
 
-    **Board 54 "Kiwi"**
-
-    Chairwoman: Marthe Meskers ([voorzitter@krashna.nl](mailto:voorzitter@krashna.nl))\
-    Secretary: Pim de Jong ([secretaris@krashna.nl](mailto:secretaris@krashna.nl))\
-    Treasurer: Duschka Kleijn ([penningmeester@krashna.nl](mailto:penningmeester@krashna.nl))\
-    Acquisition commissioner: Maurice Hooikammer ([acquisiteur@krashna.nl](mailto:acquisiteur@krashna.nl))\
-    Orchestra commissioner: Thomas Scheepmaker ([orkestcom@krashna.nl](mailto:orkestcom@krashna.nl))\
-    Choir and concert commissioner: Fos van der Meer ([koorcom@krashna.nl](mailto:koorcom@krashna.nl) and [concertcom@krashna.nl](mailto:concertcom@krashna.nl))\
-    Public relations and chamber music commissioner: Ben Carp ([publicom@krashna.nl](mailto:publicom@krashna.nl) and [kamucom@krashna.nl](mailto:kamucom@krashna.nl))
+    **Board 55 "Klimop"**
+
+    Chairwoman and chamber music commissioner: Ilja de Goede ([voorzitter@krashna.nl](mailto:voorzitter@krashna.nl) and [kamucom@krashna.nl](mailto:kamucom@krashna.nl))\
+    Secretary: Annemarijne Verschuijl ([secretaris@krashna.nl](mailto:secretaris@krashna.nl))\
+    Treasurer: Merijn Jongepier ([penningmeester@krashna.nl](mailto:penningmeester@krashna.nl))\
+    Acquisition commissioner: Mette Klein ([acquisiteur@krashna.nl](mailto:acquisiteur@krashna.nl))\
+    Orchestra commissioner: Jonah Meijer ([orkestcom@krashna.nl](mailto:orkestcom@krashna.nl))\
+    Choir commissioner: Sara Lubbers ([koorcom@krashna.nl](mailto:koorcom@krashna.nl))\
+    Public relations and concert commissioner: Sanne Peters ([publicom@krashna.nl](mailto:publicom@krashna.nl) and [kamucom@krashna.nl](mailto:concertcom@krashna.nl))
```

Comment:
kamucom needs to be updated to concertcom as well

Reply:
Oh, thanks, missed that

Reply:
Should I make a new pull request?

Reply:
No, it's already merged and deployed